src/passes/Metrics.cpp

```diff
@@ -24,7 +24,7 @@ namespace wasm {
 using namespace std;
 
 // Prints metrics between optimization passes.
-struct Metrics : public WalkerPass<PostWalker<Metrics>> {
+struct Metrics : public WalkerPass<PostWalker<Metrics, UnifiedExpressionVisitor<Metrics>>> {
```

Comment:
metrics pass is an example of a pass that wants the new UnifiedExpressionVisitor.